lib/pure/collections/sequtils.nim

```diff
@@ -140,6 +140,31 @@ func concat*[T](seqs: varargs[seq[T]]): seq[T] =
       result[i] = itm
       inc(i)
 
+func addUnique*[T](s: var seq[T], x: sink T) =
```

Comment:
Should the same exist for `string` in `strutils` ?. 🤔

Reply:
Can be added later.

Reply:
Then this needs a follow up PR.